Makefile

```diff
@@ -207,7 +207,7 @@ localunit: test/goecho/goecho varlink_generate
 	./contrib/cirrus/lib.sh.t
 
 ginkgo:
-	ginkgo -v -tags "$(BUILDTAGS)" $(GINKGOTIMEOUT) -cover -flakeAttempts 3 -progress -trace -noColor -nodes 3 -debug test/e2e/.
+	ginkgo -v -tags "$(BUILDTAGS)" $(GINKGOTIMEOUT) -cover -flakeAttempts 3 -progress -trace -noColor -debug test/e2e/.
```

Comment:
Friendly reminder to put back `-nodes 3` when appropriate (I'm not sure we're past the odd hanging condition yet or not).

Reply:
That's why it's in a standalone commit.

Reply:
kk gotcha.